concordium-consensus/src/Concordium/GlobalState/Persistent/Account/StructureV1.hs

```diff
@@ -8,6 +8,7 @@
 {-# LANGUAGE StandaloneDeriving #-}
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE UndecidableInstances #-}
```

Comment:
I didn't manage to derive the `BlobStorable` instance for `Conditionally` as the default implementation derived from its `Serialize` instance. Once this is done, this isn't needed anymore.